Wikipedia/UI-V5/WMFArticleListCollectionViewController.m

```diff
@@ -54,6 +55,7 @@ - (instancetype)initWithCoder:(NSCoder*)aDecoder {
 }
 
 - (void)commonInit {
+    [self observeArticleUpdates];
```

Comment:
Interesting... doesn't work with `viewWillAppear` / `didDisappear `. The update is happening when the view isn't visible, so I guess it makes sense. Moved it to `viewDidLoad` and it works though.